docs/source/kit/servo-board.rst

```diff
@@ -0,0 +1,69 @@
+Servo Board
+===========
+
+The Servo Board can be used to control up to 12 RC servos. Many devices
+are available that can be controlled as servos, such as RC motor speed
+controllers, and these can also be used with the board.
+
+Board Diagram
+-------------
+
+.. figure:: /_static/kit/sbv4_diagram.png
+   :alt: Board Diagram
+   :scale: 75%
+
+   Board Diagram
+
+Indicators
+----------
+
+===== ==================== ======================
+LED   Meaning              Initial power-up state
+===== ==================== ======================
+Power The board is powered On
```

Comment:
This table is incomplete, but updating it (and likely the diagram too) is an upstream issue: srobo/docs#103. I have opened #17 to track fixing this in this repo once the upstream issue is resolved.